pkg/cover/server.go

```diff
@@ -38,6 +38,7 @@ const LogFile = "goc.log"
 
 type server struct {
 	PersistenceFile string
+	NetworkType     string //solve different network type(1.regist client can direct access ,2.regist client under a proxy network or nat network、same network ect)
```

Comment:
The concept  of `NetworkType` seems too  big, actually the goal here is only to check whether to do ip revise during registering。so i prefer:
```go
type server struct{
   ...
   IPRevise bool // whether to do ip revise during registering
```

Reply:
`IPRevise` can be passed when goc server lauching like `goc server --ip_revise=false`.
Offically by default, `IPRevise` should be true to keep consistent with previous versions.